boards/ct/ctcc/doc/index.rst

```diff
@@ -114,84 +172,93 @@ logs on emulated USB port.
    :board: ctcc/nrf52840
    :goals: build flash
 
-Debugging
-=========
+Option 2: Using MCUboot with MCUmgr support
+-------------------------------------------
 
-The ``ctcc/nrf52840`` board target does not have an on-board J-Link debug IC, however
-instructions from the :ref:`nordic_segger` page also apply to this board,
-with the additional step of connecting an external debugger.
+It is also possible to use the MCUboot bootloader with :ref:`mcu_mgr` support to flash
+Zephyr applications.
+
+Install a MCUmgr-compatible tool from :ref:`supported list <mcumgr_tools_libraries>`
+and make sure MCUboot's ``imgtool`` is available for signing your binary
+for MCUboot as described on :ref:`west-sign`.
+
+#. Compile MCUboot as a Zephyr application with ``MCUmgr`` support.
 
-Option 2: Using MCUboot with DFU support
-----------------------------------------
+.. tabs::
```

Comment:
this entire `.. tabs` sections needs to be indented by 3 spaces to correctly fall under the first numbered list item.

Reply:
thanks, fixed

Reply:
thanks, better! https://builds.zephyrproject.io/zephyr/pr/82814/docs/boards/ct/ctcc/doc/index.html#option-2-using-mcuboot-with-mcumgr-support :)